src/main/java/org/jenkinsci/test/acceptance/po/WorkflowJob.java

```diff
@@ -37,4 +38,9 @@ public WorkflowJob(Injector injector, URL url, String name) {
     public final Control script = control("/definition/script");
     public final Control sandbox = control("/definition/sandbox");
 
+    public String copyResourceStep(String filePath) {
+        final Resource res = resource(filePath);
+        return String.format("sh '''%s'''%n", copyResourceShell(res, res.getName()));
```

Comment:
I think this is OK, just be careful, since `'''` does not escape everything—in particular `'''` itself (obviously), but also `\` (less obviously). So long as the return value of `copyResourceShell` never includes either of these things it should work.